apps/fz_common/lib/fz_net.ex

```diff
@@ -48,6 +48,21 @@ defmodule FzCommon.FzNet do
     end
   end
 
+  def inet_to_ip_with_mask(%{address: address, netmask: netmask} = inet, network) do
+    if netmask != nil do
+      inet
+    else
+      "#{address |> Tuple.to_list() |> Enum.join(".")}/#{get_range!(network)}"
```

Comment:
This will make this function always return a string and make the code a bit more readable. 
```suggestion
      EctoNetwork.CIDR.decode(inet)
    else
      address = address |> Tuple.to_list() |> Enum.join(".")
      "#{address}/#{get_range!(network)}"
```

But is there a case when `inet.netmask` won't be `nil`? If it's not nil then it's not an inet, but a cidr, right? Maybe we can just remove the if condition entirely and keep the `else` block.

Reply:
Hmm does the [EctoNetwork.INET module](https://github.com/adam12/ecto_network/blob/v1.3.0/lib/ecto_network/inet.ex#L22) already have functions for handling this logic?

Reply:
Looks like [`EctoNetwork.INET.decode/1`](https://github.com/adam12/ecto_network/blob/v1.3.0/lib/ecto_network/inet.ex#L81) may already handle this for us?

Reply:
I don't know why I wrote it like this, used `:inet.ntoa` instead.

> But is there a case when inet.netmask won't be nil? If it's not nil then it's not an inet, but a cidr, right? Maybe we can just remove the if condition entirely and keep the else block.

The idea of this function is that a user can write for the firezone wireguard interface the ip like `100.64.0.1/10`, in that case we believe the netmask that they used, but in other cases they could write `100.64.0.1` in that case we assume the ip to be part of the wireguard network, therefore we use the network's netmask.

Reply:
I'm changing the configuration to make sure they can't put invalid CIDR(like an ip without a range) in network fields, so this issue will go away completely.

Reply:
oh yeah, that'd remove the need for this function completely.